ext/debug/segger/Kconfig

```diff
@@ -14,3 +14,45 @@ config SEGGER_SYSTEMVIEW
 	default n
 	depends on HAS_SEGGER_RTT
 	select RTT_CONSOLE
+
+config SEGGER_RTT_MAX_NUM_UP_BUFFERS
```

Comment:
It is possible to avoid repetition of 'depends on HAS_SEGGER_RTT' for every entry you added, wrap all entries in `if  HAS_SEGGER_RTT ... endif`  - Do this in your convenience.

Reply:
Agreed. It looks like you made this change, but in the wrong commit. The Kconfig changes should go in the previous commit, not the commit updating RTT to 6.32d.